Gemfile

```diff
@@ -15,8 +14,7 @@ gem 'whenever'
 
 gem 'browser'
 gem 'ckeditor_rails'
-gem 'formtastic', '~>2.3.0'
-gem 'foundation-rails'
+gem 'foundation-rails', '~> 6.3.1.0'
```

Comment:
6.3.1 is a bit buggy - I find 6.4.1.2 is the most reliable version available at the moment but NZSL doesn't use any of the Foundation fancy things so this is 100% 🐘